inc/dashboard/grid.class.php

```diff
@@ -957,6 +957,10 @@ public function getCardHtml(string $card_id = "", array $card_options = []): str
       }
       $card  = $cards[$card_id];
 
+      if (isset($card['keywords'])) {
```

Comment:
Unless I am wrong `keywords` is specific to `advanceddashboard` plugin. I would prefer to add a generic logic, for instance by checking a `custom_hash` key and use it instead of `sha1(serialize($card_options)`.
It would permit to each plugin to easilly control uniqueness of its cards in cache system.

Also, as it is done to handle plugins issues, maybe a comment in code could be usefull.